### ISSUE TITLE
git:// schema not detected

### ISSUE THREAD

**Issue by HackingRepo** [coreruleset/coreruleset#4196]
Hi, Team

I detect today the git:// schema not blocked again can be lead to SSRF (Git Internal daemon abuse) or even RCE in some case in PL3
```bash
curl -ig -H "x-format-output: txt-matched-rules" -H "x-crs-paranoia-level: 3" -H "x-backend: coraza-caddy" --data-urlencode "git://evil.attacker.com/repo.git" "https://sandbox.coreruleset.org/"
```

```
HTTP/1.1 200 OK
Date: Thu, 10 Jul 2025 07:29:22 GMT
Content-Type: text/plain
Transfer-Encoding: chunked
Connection: keep-alive
X-Unique-ID: aG9r0g3ZsEJGA-19Uz89VwAAANQ
x-backend: invalid, fallback to apache-latest
x-crs-last-commit: none
```

**The WAF Should block git:// protocol at PL 3 sometime can be abused**

**Comment by airween** [comment]
Again: your query is not valid. There is no name/value for the request argument. With a valid request, CRS blocks this request on PL2.

**Please-please-please be more careful and attentive**! With these kind of issues you're just wasting our time.
